ios/FluentUI/Navigation/Views/AvatarTitleView.swift

```diff
@@ -322,7 +322,9 @@ class AvatarTitleView: UIView, TokenizedControlInternal, TwoLineTitleViewDelegat
         twoLineTitleView.setup(navigationItem: navigationItem)
 
         // Hide the avatar if TwoLineTitleView has a leading image for both title and subtitle.
-        showsProfileButton = !navigationItem.isTitleImageLeadingForTitleAndSubtitle
+        if navigationItem.isTitleImageLeadingForTitleAndSubtitle {
+            showsProfileButton = false
```

Comment:
Would we ever need to set it to true again?

Reply:
good point. I think we will need to recalculate the visibility of the button for the "else" scenario. I have updated to reflect that.